x-pack/plugins/alerting/server/task_runner/create_execution_handler.ts

```diff
@@ -223,6 +224,7 @@ export function createExecutionHandler<
             },
             { type: 'action', id: action.id, type_id: action.actionTypeId, ...namespace },
           ],
+          version: '1.0.1',
```

Comment:
I think this should be set to `kibanaVersion` instead of hard-coded?

Reply:
ops :)